lesson-5.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+    <link rel="stylesheet" href="css/lesson-5.css">
+    <div class="container_about">
+        <p class="heading_about">About</p>
+        <p class="text">Now Triphouse works</p>
```

Comment:
"text" classname doesn't display content. you can have a lot of text elements on the site and they will be different.
the name of the classes should store information about the content